Dockerfile

```diff
@@ -1,5 +1,11 @@
 # Build the manager binary
-FROM golang:1.16 as builder
+## Base image for general public:
+# FROM golang:1.16 as builder
+## or
+# FROM mirror.gcr.io/library/golang:1.16 as builder
+## Base image for Github CI:
+FROM openshift/origin-release:golang-1.16 as builder
```

Comment:
why not 1.17 ?

Reply:
Done with `mirror.gcr.io/library/golang:1.17`.
All the openshift/origin-release options are failing. 
BTW, it looks like there is no official openshift  golang:1.17 yet - tried to pull it manually got 404.